libraries/AP_HAL_Linux/Util.h

```diff
@@ -111,7 +152,9 @@ class Util : public AP_HAL::Util {
     const char *custom_storage_directory = nullptr;
     const char *custom_defaults = HAL_PARAM_DEFAULTS_PATH;
     static const char *_hw_names[UTIL_NUM_HARDWARES];
-
+    int _affinity_core_num = 0;  // '0' means ignore affinity
```

Comment:
I think it may actually be valid to use core 0, depending on your board and kernel config. Example: you can use isolcpus in the kernel command line and then I think then you can isolate core 0 an then be able to use cpu affinity to move threads there.

Reply:
Now cpu bitmask can use any available cpu.